src/main/java/com/impactupgrade/nucleus/service/segment/MailchimpCommunicationService.java

```diff
@@ -162,13 +160,18 @@ protected void syncContacts(PagedResults.ResultSet<CrmContact> resultSet, Enviro
 
       String archiveBatchId = mailchimpClient.archiveContactsBatch(communicationList.id, emailsToArchive);
       mailchimpClient.runBatchOperations(mailchimpConfig, archiveBatchId, 0);
+
+      List<MemberInfo> memberInfos = mailchimpClient.getListMembers(communicationList.id);
+      backfillContacts(contactsToUpsert, memberInfos);
     } catch (MailchimpException e) {
       env.logJobWarn("Mailchimp syncContacts failed: {}", mailchimpClient.exceptionToString(e));
     } catch (Exception e) {
       env.logJobWarn("Mailchimp syncContacts failed", e);
     }
   }
 
+  protected void backfillContacts(List<CrmContact> contacts, List<MemberInfo> memberInfos) {}
```

Comment:
@VSydor We can remove this method and no need to call it from `syncContacts`. For the backfill, let's take the code you put together in dr-hub (the implementation of this method) and we'll instead do a one-time script in `ManualUtil`. The backfill is just a one-time thing to catch MC contacts up. After that, it's just the normal daily sync as contacts update in SFDC.